autowebcompat/network.py

```diff
@@ -166,3 +166,13 @@ def compile(model, optimizer='sgd', loss_func=contrastive_loss):
     opt = allOptimizers[optimizer]
 
     model.compile(loss=loss_func, optimizer=opt, metrics=[accuracy])
+
+
+def supported_network():
```

Comment:
Define these at the beginning of the file. Also, they don't need to be functions but can be lists directly:
```Python
SUPPORTED_NETWORKS = ['inception', 'vgglike', 'vgg16']
```